src/Command/Playercount.js

```diff
@@ -0,0 +1,73 @@
+const Command = require('../Command');
+
+const messages = {
+    'errorChecking': 'Hmm.. I encountered some issues looking up the players.  Is Shotbow.net offline?',
+    'errorBadKey': 'Hmm.. I couldn\'t find the game you were talking about.  Try again?',
+    'result': 'There are currently %1 players connected to %2',
+};
+
+const keyAlias = {
+    'hcf': 'hcfactions',
+    'anni': 'annihilation',
+    'civ-war': 'civwar',
+    'civcraft': 'civwar',
+    'minetheftauto': 'mta',
+    'gc': 'ghostcraft'
+};
+
+// Map of key to name - also acts as a whitelist
+const gameNames = {
+    'all': 'Shotbow',
+    'annihilation': 'Annihilation',
+    'minez': 'MineZ',
+    'hcfactions': 'HCFactions',
+    'dbv': 'DBV',
+    'smash': 'SMASH',
+    'slaughter': 'Slaughter',
+    'civwar': 'Civ-War',
+    'gg': 'gg',
+    'mta': 'Mine Theft Auto',
+    'ghostcraft': 'GhostCraft',
+};
+
+module.exports = Command.extend({
+    commandName: 'playercount',
+    https: null,
+    dependencies: {
+        'discordClient': 'discordClient',
+        'commandPrefix': 'commandPrefix',
+        'https': 'https',
+    },
+    processMessage: function (message, tokens) {
+        this.https.get('https://shotbow.net/serverList.json', res => {
```

Comment:
@JamsheedMistri Is there a case here for if the connection fails I should be listening for?